poetry.lock

```diff
@@ -1533,7 +1537,7 @@ python-versions = "*"
 
 [[package]]
 name = "wrapt"
-version = "1.13.3"
+version = "1.14.1"
```

Comment:
Old version was using a deprecated function which was removed in Python 3.11, see https://github.com/GrahamDumpleton/wrapt/pull/205